src/txn-wal/src/operator.rs

```diff
@@ -667,8 +669,8 @@ impl DataSubscribe {
                 (data_stream.leave(), token)
             });
             let (data, txns) = (ProbeHandle::new(), ProbeHandle::new());
-            let data_stream = data_stream.flat_map(|part| {
-                let part = part.parse();
+            let data_stream = data_stream.flat_map(move |part| {
+                let part = part.parse(persist_cfg.clone());
```

Comment:
this is all pretty gross. I wonder if it would make more sense to have the public parse just take a bool for validation enabled?